source/user-manual/kibana-app/reference/config-file.rst

```diff
@@ -207,6 +259,79 @@ Configure wazuh-monitoring-* indices custom creation interval.
 | **Allowed values** | h (hourly), d (daily), w (weekly), m (monthly) |
 +--------------------+------------------------------------------------+
 
+Cron
+----
+
+cron.prefix
+^^^^^^^^^^^
+
+Customize the index prefix of predefined jobs. This change is not retroactive, if you change it new indexes will be created
+
++--------------------+----------------------+
+| **Default value**  | wazuh                |
++--------------------+----------------------+
+| **Allowed values** | String. Eg: ["test”] |
++--------------------+----------------------+
+
+Statistics
+----------
+
+cron.statistics.status
+^^^^^^^^^^^^^^^^^^^^^^
+
+Custom setting to enable/disable statistics tasks
+
++--------------------+------------+
+| **Default value**  | true       |
++--------------------+------------+
+| **Allowed values** | true,false |
++--------------------+------------+
+
+cron.statistics.apis
+^^^^^^^^^^^^^^^^^^^^
+
+Enter the ID of the APIs you want to save data from, leave this empty to run
+the task on all configured APIs
+
++--------------------+----+
+| **Default value**  | [] |
++--------------------+----+
+| **Allowed values** | [] |
++--------------------+----+
+
+cron.statistics.interval
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Define the frequency of task execution using cron schedule expressions.
+
++--------------------+----------------------------+
+| **Default value**  | 0                          |
++--------------------+----------------------------+
+| **Allowed values** | Any number starting from 0 |
```

Comment:
The default value is `0 */5 * * * *` and the allowed value is a cron schedule expression: https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm